src/sankey.js

```diff
@@ -1,7 +1,21 @@
 import {max, min, sum} from "d3-array";
 import {justify} from "./align.js";
+import {transformTop, transformRight, transformBottom, transformLeft} from "./transform.js";
+import {sankeyLinkHorizontal, sankeyLinkVertical} from "./sankeyLink.js";
 import constant from "./constant.js";
 
+var top = 1,
```

Comment:
Suggest consistent use of `let` and `const`, preferring `const` where possible. Currently there are instances `var`, `let` and `const` here.
```suggestion
const top = 1,
```